.github/workflows/publishondockerhub.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Publish
+on:
+  push:
+    branches:
+      - 'parachain_dev'
+    tags:
+      - 'peaq-dev-v*' # For the peaq-dev's env in the parchain-launch usage
+      - 'krest-v*' # For the krest env in the parchain-launch usage
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Free Disk Space (insightsengineering/disk-space-reclaimer)"
+        uses: insightsengineering/disk-space-reclaimer@v1.1.0
+
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ${{ secrets.DOCKER_REG_PARACHAIN}}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=ref,event=tag
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: scripts/Dockerfile
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Extract Tag
+        id: image-version
+        run: |
+          input_string=${{ steps.meta.outputs.tags}}
+          tag=$(echo "$input_string" | cut -d ':' -f 2)
+          echo "Extracted Tag: $tag"
+           echo "::set-output name=tag::$tag"
+           
+      - name: executing remote ssh commands using ssh key
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ${{ secrets.AUTOTEST_HOST }}
+          username: ${{ secrets.AUTOTEST_USERNAME }}
+          key: ${{ secrets.SSH_KEY_AUTOTEST }}
+          port: ${{ secrets.AUTOTEST_SSH_PORT }}
+          script: |
+            cd /home/jay/parachain-launch/yoyo/
+            echo ${{ steps.extract.outputs.tag }}
+            sudo TAG=${{ steps.image-version.outputs.tag }} docker compose down -v
```

Comment:
Will we fetch the docker image?

Reply:
yes, passing docker image version in TAG and  using TAG as arg in parachain-2000.Dockerfile

Reply:
also changed base branch to parachain_dev